src/Components/Users_List.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState, useEffect} from 'react';
+import UsersChart from './Users_Chart';
+import styles from './css/style.css';
+
+const Users_List = (props) => {
+    const user = props.userList;
+    const [currentUser, setCurrentUser] = useState(0);
+
+    useEffect(() => {
+        setCurrentUser(0);
+    }, [user]);
+
+    return (
+        <div className="div-usersList">
+            <ul>
+                {user.map((el, index) => 
+                <li 
+                    onClick={() => { setCurrentUser(index) }} 
+                    key={index}>{el.firstName} {el.lastName}

```

Comment:
if you have an id available in the item you are displaying, always prefer that instead of the array index. Index should be the last choice